app/models/workspace.js

```diff
@@ -0,0 +1,33 @@
+import DS from 'ember-data';
+import Ember from 'ember';
+import FolderItemMixin from 'wholetale/mixins/folder-item';
+import AccessControlMixin from 'wholetale/mixins/access-control';
+
+export default DS.Model.extend(FolderItemMixin, AccessControlMixin, {
+  _accessLevel: DS.attr(),
+  _id: DS.attr(),
+  _modelType: DS.attr('string'),
+  baseParentType: DS.attr('string'), // folder or item
+  baseParentId: DS.attr('string'), // folder or item
+  // baseParentId: Ember.computed('baseParentType', function() {
```

Comment:
This class was copied from the existing model for `folder`.. I was unsure what the commented out fields on that model were for, but preserved them here in case they were desirable.

I'd be happy to replace them with a shorter comment referencing the original implementation